src/view/com/composer/Composer.tsx

```diff
@@ -571,6 +567,7 @@ export const ComposePost = ({
                       dispatch({type: 'update_labels', labels: nextLabels})
                     }}
                     hasMedia={hasMedia || Boolean(extLink)}
+                    Portal={Portal.Portal}
```

Comment:
We don't need this right? Is it still in the types?

Reply:
nope, lemme rebase and remove